compose-combined.yml

```diff
@@ -0,0 +1,29 @@
+services:
```

Comment:
Can you please add a documentation to README.md in the Deployment section on how to deploy using a single docker image?

Reply:
I'm not sure how to deploy yet.
I'm not happy with needing to forward 2 ports to use 1 container and to have to provide the API address at build time.

Is it not possible to change something so that the API Address is not required at build time? 
In this "combined" case the hostname will likely always be the same as the one that is in the "browser address bar" 
The API calls could be made to `<hostname>:<port>/api` which then can be proxied to the 8080 port internally with nginx for example?